api/namex/resources/requests.py

```diff
@@ -531,6 +531,9 @@ def patch(nr, *args, **kwargs):
                             new_comment.nrId = nrd.id
 
                 ### END comments ###
+            elif consume := json_input.get('consume', None):
```

Comment:
@thorwolpert do i need to do any state change for nr?

Reply:
no, that state doesn't change.
This will set the consumed info in the names DB, which then gets synced back to NameX.